tests/mlio-py-test/test_data_reader_params.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
Please run `flake8` and address the (minor) styling issues. While you are at it, it would be great if you could also remove the dummy `test_foo.py`?